calyx-py/test/correctness/queues/strict_and_rr_queues/strict_queues/strict_6flows.py

```diff
@@ -0,0 +1,13 @@
+import os
+import sys
+import inspect
+
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
```

Comment:
These ugly imports are a little unfortunate but we'll live with them until #2186 lands!